src/Blocks/components/form/assets/form.js

```diff
@@ -729,6 +729,21 @@ export class Form {
 					break;
 			}
 		}
+
+		// Used in case we have req fields with conditional tags and multiple steps.
+		if (skipFields) {
+			const skipFieldsOutput = {};
+			skipFields.forEach((skipField) => {
+				skipFieldsOutput[skipField] = this.state.getStateElementValue(skipField, formId);
+			});
```

Comment:
```suggestion
			const skipFieldsOutput = skipFields.reduce((previous, skipField) => ({
				...previous,
				[skipField]: this.state.getStateElementValue(skipField, formId),
			}), {});
```

Reply:
I really don't like the semantics of reduce. It is hard to read and understand. Tnx for your suggestion but I will skip this, tnx